docs/setup.rst

```diff
@@ -124,9 +126,22 @@ plugins          A list of plugins to load (see :ref:`plugin-config`)
 .. [#unset] That is to say, the option is simply unset by default, the
     equivalent of not mentioning the option at all in a configuration file.
 
-.. [#plugins] This option can *only* appear in the configuration file; there is
+.. [#plugins_config] This option can *only* appear in the configuration file; there is
```

Comment:
It gets rid of a document build warning.

On Fri, Jan 22, 2016 at 11:50 AM, Roni Choudhury <notifications@github.com>
wrote:

> In docs/setup.rst
> <https://github.com/Kitware/tangelo/pull/542#discussion_r50560208>:
>
> > @@ -124,9 +126,22 @@ plugins          A list of plugins to load (see :ref:`plugin-config`)
> >  .. [#unset] That is to say, the option is simply unset by default, the
> >      equivalent of not mentioning the option at all in a configuration file.
> >
> > -.. [#plugins] This option can *only* appear in the configuration file; there is
> > +.. [#plugins_config] This option can *only* appear in the configuration file; there is
>
> Why this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/tangelo/pull/542/files#r50560208>.
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439